tools/scaffolding/templates/main.tf.template

```diff
@@ -15,7 +15,7 @@ provider "aws" {
 
 
 module "db_instance" {
-  source = "git::https://github.com/dfds/terraform-aws-rds.git?ref=<release_number>"
+  source = "git::https://github.com/dfds/terraform-aws-rds.git?ref=0.0.1"
```

Comment:
We are already at 0.16.0

Reply:
I know but I want to check if renovate can act on it.